client/state/selectors/get-checkout-incompatible-products.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { translate, TranslateResult } from 'i18n-calypso';
```

Comment:
Perhaps instead of in this location, this code can be part of this file:

https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/products/conflicts.ts

Reply:
Awesome idea. I struggled a lot with where to put this code.

Reply:
Done!